components/combobox/index.css

```diff
@@ -0,0 +1,451 @@
+      border-color: var(--highcontrast-combobox-border-color, var(--mod-combobox-border-color-default, var(--spectrum-alias-infieldbutton-fill-border-color-default)));
+    }
+
+    &:focus, &.is-focused, &.is-keyboardFocused {
+      .spectrum-PickerButton-fill:not(.is-invalid) {
+        border-color: var(--highcontrast-combobox-bordr-color, var(--mod-combobox--border-color-key-focus, var(--spectrum-alias-infieldbutton-fill-border-color-key-focus)));
+      }
+    }
+  }
+
+  & .is-open.spectrum-Popover {
+    transform: translateY(var(--mod-combobox-spacing-edge-to-menu, var(--spectrum-combobox-spacing-edge-to-menu)));
+  }
+}
+
+.spectrum-Combobox.is-invalid {
+  .spectrum-Combobox-input {
+    border-color: var(--highcontrast-combobox-border-color, var(--mod-combobox-border-color-invalid-default, var(--spectrum-combobox-border-color-invalid-default)));
+
+    &:hover {
+      border-color: var(--highcontrast-combobox-border, var(--mod-combobox-border-color-invalid-hover, var(--spectrum-combobox-border-color-invalid-hover)));
+    }
+
+    &:active {
+      border-color: var(--highcontrast-combobox-border, var(--mod-combobox-border-color-invalid-active, var(--spectrum-combobox-border-color-invalid-active)));
+    }
+
+    &.focus-ring {
+      box-shadow: none;
+      /* Subcomponent token override */
+      --spectrum-alias-infieldbutton-fill-border-color-key-focus: var(--spectrum-combobox-border-color-invalid-key-focus);
+      border-color: var(--highcontrast-combobox-border-color, var(--mod-combobox-border-color-key-focus, var(--spectrum-alias-infieldbutton-fill-border-color-key-focus)));
+    }
+  }
+
+  .spectrum-Textfield-validationIcon {
+    color: var(--highcontrast-combobox-icon-color, var(--mod-combobox-alert-icon-color, var(--spectrum-combobox-alert-icon-color)));
+  }
+}
+
+.spectrum-Combobox.is-disabled {
+  .spectrum-Combobox-input {
+    background-color: var(--highcontrast-combobox-background-color, var(--mod-combobox-background-color-disabled, var(--spectrum-combobox-background-color-disabled)));
+
+    &::placeholder {
+      color: var(--highcontrast-combobox-font-color-disabled, var(--mod-combobox-font-color-disabled, var(--spectrum-combobox-font-color-disabled)));
+    }
+  }
+}
+
+.spectrum-Combobox-progress-circle {
+  position: absolute;
+  inset-inline-end: calc(
+    var(--mod-combobox-spacing-inline-icon-to-button, var(--spectrum-combobox-spacing-inline-icon-to-button)) +
+    var(--mod-combobox-button-size, var(--spectrum-combobox-button-size))
+  );
+  inset-block-start: var(--mod-combobox-block-spacing-edge-to-progress-circle, var(--spectrum-combobox-block-spacing-edge-to-progress-circle));
+  inset-block-end: var(--mod-combobox-block-spacing-edge-to-alert, var(--spectrum-combobox-block-spacing-edge-to-alert));
+}
+
+/* Overrides inset-inline position from spectrum-Textfield.is-invalid */
+[dir="ltr"] .spectrum-Textfield.is-invalid {
```

Comment:
I think the Combobox button also needs to be swapped on the side for LTR support